### DISCUSSION TITLE
429 Too Many Requests error on Payload API

### DISCUSSION THREAD

**Discussion by elvis-chong-innoqb** [payloadcms/payload#9656] [Q&A] [answered]
I’m currently using Payload version 2.30.1 and have developed a custom module for data migration. This module processes a large dataset containing over 1,000 records. However, I’ve encountered an issue where, after migrating around 200+ records, the API starts returning a 429 Too Many Requests error.

This is happening even though I’m running the application on localhost and haven’t enabled any rate limit configuration in my server.js. I was wondering if there is a way to turn off or bypass the rate limit for the Payload API during data migration.

Any guidance or suggestions on how to resolve this would be greatly appreciated.

![image](https://github.com/user-attachments/assets/d297c625-65dc-4d26-af81-bcc39affd2da)

**Comment by matthijs166** [comment, marked as answer]
There is a rate limit set in the default config:
https://payloadcms.com/docs/configuration/overview#config-options
See the `rateLimit` property on the payload config.
